gpustack/routes/model_instances.py

```diff
@@ -82,7 +82,7 @@ async def get_serving_logs(
     server_config: Config = request.app.state.server_config
 
     model_instance_log_url = (
-        f"http://{worker.ip}:{server_config.worker_port}/serveLogs"
+        f"http://{worker.ip}:{worker.port}/serveLogs"
```

Comment:
Remove unused `server_config` variable to address the CI linting issue.

Reply:
already fix and CI lint issue was sloved